src/server.py

```
@@ -1,26 +1,53 @@
+
 """Server Module that recives message and sends back to client."""
 import socket
 
 
 def server():
-    """Hi."""
+    """The server module receives and sends a message."""
     server = socket.socket(socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_TCP)
-    address = ('127.0.0.1', 5036)
+    address = ('127.0.0.1', 5069)
     server.bind(address)
     server.listen(1)
     while True:
         try:
             conn, addr = server.accept()
             buffer_length = 15
             message_complete = False
-            response = ""
+            request = ""
             while not message_complete:
                 part = conn.recv(buffer_length)
-                response += part.decode('utf8')
+                request += part.decode('utf8')
                 if len(part) < buffer_length:
                     break
-            conn.sendall(response.encode('utf8'))
-            print(response)
+            parse_request(request)
+            ok = response_ok()
+            conn.sendall(ok.encode('utf8'))
+            print(request)
             conn.close()
         except KeyboardInterrupt:
+            response_error()
             server.close()
+
+
+def parse_request(request):
+    """Parse message request from client and return error if necessary."""
+    parsed = request.split('\r\n')
+    if parsed[0][:3] != "GET":
+        raise ValueError("Error 405: Can only use GET method.")
```

These should not be 'raised' in the normal python sense.  It crashes your server.  You need to return an appropriate error message to the client.